lang/main.json

```diff
@@ -643,5 +644,28 @@
         "rejected": "Rejected",
         "ignored": "Ignored",
         "expired": "Expired"
+    },
+    "localRecording": {
+        "localRecording": "Local Recording",
+        "dialogTitle": "Local Recording Controls",
+        "start": "Start",
+        "stop": "Stop",
+        "moderator": "Moderator",
+        "localUser": "Local user",
```

Comment:
For consistency, I'd try to get verification on strings whenever verification on design is ever obtained. I don't think in the UI the local participant is referred to as local user

Reply:
Should it be "Me" then?